test/sequential/test-http-regr-gh-2928.js

```diff
@@ -18,13 +18,26 @@ var gotRequests = 0;
 var gotResponses = 0;
 
 function execAndClose() {
-  process.stdout.write('.');
   if (parsers.length === 0)
     return;
+  process.stdout.write('.');
 
   const parser = parsers.pop();
   parser.reinitialize(HTTPParser.RESPONSE);
+
   const socket = net.connect(common.PORT);
+  socket.on('error', (e) => {
+    // If SmartOS and ECONNREFUSED, then retry. See
+    // https://github.com/nodejs/node/issues/2663.
+    if (common.isSunOS && (e.code === 'ECONNREFUSED')) {
```

Comment:
> Please skip additional parens here.

Additional parens removed, branch rebased, force pushed. Will run CI one more time.

There's a build-in lint rule to catch extra parentheses,  Alas, enabling it flags 315 instances in the code base. Probably not worth the churn. I might look more closely at it, though...